git_machete/client.py

```diff
@@ -1906,8 +1906,9 @@ def __sync_before_creating_pr(self) -> None:
         if current_branch not in self.managed_branches:
             self.add(current_branch)
 
-        if self.__is_merged_to_upstream(current_branch):
-            return
+        if self.__git.is_ancestor_or_equal(current_branch, self.up_branch.get(current_branch), equal_only=True):
```

Comment:
Hmmm okay but if current branch points to an **ancestor** commit of parent branch... will Github agree to open a PR from current branch to parent branch?

Reply:
Yes, you were right, Github doesn't allow to open pr under such circumstances. So, i changed the condition now, opening a PR is allowed only if head branch has at least one commit which is not included in base branch.